nullaway/src/main/java/com/uber/nullaway/NullabilityUtil.java

```diff
@@ -54,6 +55,10 @@
 
 /** Helpful utility methods for nullability analysis. */
 public class NullabilityUtil {
+  public static final Class<? extends Annotation> NULLMARKED =
+      org.jspecify.nullness.NullMarked.class;
+
+  private static final NullMarkedCache nullMarkedCache = new NullMarkedCache();
```

Comment:
I am a bit concerned about making this cache static.  Even though the size is limited, since a `ClassSymbol` is part of a cache entry, it's possible a lot of other `javac` state will be retained as well if it is reachable from the `ClassSymbol`, which could lead to a large negative impact on memory footprint.  Plus there is the question of thread safety for concurrent builds.  How painful would it be to make the cache non-static?  Unfortunately it seems it might be a bit of a pain.

Reply:
Unfortunately, fairly hard, as this needs to be threaded not only through the standard NullAway AST traversal, but through the full dataflow transfer functions code as well... even in parts we aren't threading the `Config` object currently. I can take a look into doing that, but it will probably be fairly involved...

Reply:
> That said, I do also worry about this being static, particularly in build systems that re-use `javac` instances.

Yes, this is exactly the scenario I am worried about.  I will think more to see if I can come up with an alternate solution (though nothing is coming to mind at the moment).